src/metadata_management.cc

```diff
@@ -975,4 +975,80 @@ void DecrementRefcount(SharedMemoryContext *context, RpcContext *rpc,
   }
 }
 
+u32 GetRelativeNodeId(RpcContext *rpc, int offset) {
+  int result = rpc->node_id + offset;
+  CHECK_GE(result, 0);
```

Comment:
I believe so. Since this is an internal developer interface, the check is here to prevent developer errors. In that case, I prefer to fail as fast as possible. Maybe an `assert` would be better since it's intended as a developer check.

Reply:
I changed it to `assert`.

Reply:
Thanks!